src/components/Amalgamation/AmalgamatingBusinesses.vue

```diff
@@ -165,80 +177,66 @@ export default class AmalgamatingBusinesses extends Mixins(CommonMixin) {
   // Local properties
   initialBusinessLookupObject = EmptyBusinessLookup
   businessTableValid = false
+  snackbar = false
+  snackbarText = ''
 
   // Button properties
   isAddingAmalgamatingBusiness = false
   isAddingAmalgamatingForeignBusiness = false
 
-  readonly BusinessLookupServices = BusinessLookupServices
-
-  // Cancel button in "Add an Amalgamating Business" is pressed.
-  addAmalgamatingBusinessCancel (): void {
-    this.isAddingAmalgamatingBusiness = false
-    this.setAmalgamatingBusinessesValid(true)
```

Comment:
No need to set validity here -- the watcher on line 246 handles it automatically when the variables change.

Reply:
The watcher works fine, but nothing is done with `getAmalgamatingBusinessesValid` yet -- future ticket.